exe/floe

```diff
@@ -6,16 +6,33 @@ require "optimist"
 
 opts = Optimist.options do
   version "v#{Floe::VERSION}\n"
-  opt :workflow, "Path to your workflow json", :type => :string, :required => true
-  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => '{}'
-  opt :credentials, "JSON payload with credentials", :type => :string, :default => '{}'
+  opt :workflow, "Path to your workflow json",                 :type => :string, :required => true
+  opt :inputs, "JSON payload to input to the workflow",        :default => "{}"
+  opt :credentials, "JSON payload with credentials",           :default => "{}"
+  opt :docker_runner, "Type of runner for docker images",      :default => "docker"
+  opt :docker_runner_options, "Options to pass to the runner", :type => :strings
 end
 
+Optimist.die(:docker_runner, "must be one of #{Floe::Workflow::Runner::TYPES.join(", ")}") unless Floe::Workflow::Runner::TYPES.include?(opts[:docker_runner])
+
 require "logger"
 Floe.logger = Logger.new(STDOUT)
 
 workflow = Floe::Workflow.load(opts[:workflow], opts[:inputs], opts[:credentials])
 
+runner_klass = case opts[:docker_runner]
+               when "docker"
+                 Floe::Workflow::Runner::Docker
+               when "podman"
+                 Floe::Workflow::Runner::Podman
+               when "kubernetes"
+                 Floe::Workflow::Runner::Kubernetes
+               end
+
+runner_options = opts[:docker_runner_options].to_h { |opt| opt.split("=") }
```

Comment:
I believe we expect only the first `=` to be the delimiter, and any subsequent ones would be part of the value, so I think this is more accurate:

```suggestion
runner_options = opts[:docker_runner_options].to_h { |opt| opt.split("=", 2) }
```

Reply:
Might be useful to have a spec for this case too

Reply:
@agrare If possible, please add a spec for this case.

Reply:
Oh nvm, I thought the runner spec was doing a full run using the CLI, but it's only stubbing the underlying kubernetes call and then calling fro the runner class directly.  Since we don't have specs for the `floe` script, I can merge now and we can include that in future specs for that executable.